15 - LocalStorage/index-START.html

```diff
@@ -26,13 +26,58 @@ <h2>LOCAL TAPAS</h2>
   </div>
 
 <script>
-  const addItems = document.querySelector('.add-items');
-  const itemsList = document.querySelector('.plates');
-  const items = [];
+  class customStoredList {
+    constructor(wrapperClass) {
+      this.wrapper = document.querySelector(wrapperClass);
+      this.addItems = this.wrapper.querySelector('.add-items');
+      this.itemsList = this.wrapper.querySelector('.plates');
+      this.savedItems = localStorage.getItem('items');
```

Comment:
food for thought: from the naming of the variables, I can not infer if we are talking about dom elements or data items. Is this a problem? would a different naming schema help? if yes, when do you think it makes sense to use and when is it too much?

Reply:
I get your point. It is a bit difficult to quickly differentiate the variables between DOM and Data elements. In this case I do not think it is too bad because the code is short and easy. But yes, I'm naming 3 variables using `items` as part of the name. A bit confusing.

Wes actually uses `plates` instead of `items` for the data items. Which helps a bit but I did not like it, I wanted to keep it a bit more abstract.

I think indeed it would help to use a naming schema to differentiate them. Something like the CONSTANTS for the DOM elements and normal camelCase for data variables. It would help for sure in a case like this, where you have similar names. I do not think it is ever too much. If you have a best-practice why not to use it always.

Reply:
unfortunately, i do not have a best practice. i sometimes use xxxElement or xxxElm, sometimes I don’t.
i’d not use the casing somehow. this is reserved for the type of variable/method/constant in my head. but this might be a stupid habit.